src/entities/addLiquidityNested/index.ts

```diff
@@ -33,6 +33,9 @@ export class AddLiquidityNested {
                 return addLiquidity.query(input, nestedPoolState);
             }
             case 3: {
+                if (input.chainId === 43114) {
```

Comment:
Maybe consider using `ChainId.AVALANCHE` instead of number for improved readability?

Reply:
Good suggestion, added